appserver/ejb/ejb-container/src/main/java/org/glassfish/ejb/startup/EjbDeployer.java

```diff
@@ -469,7 +468,8 @@ public void event(Event event) {
             }
 
             boolean createTimers = true;
-            boolean isDeploymentGroup = domain.getDeploymentGroupNamed(dcp.target) != null;
+            // target could be null for internal apps such as wstx-services or admin console
+            boolean isDeploymentGroup = dcp.target == null ? false : domain.getDeploymentGroupNamed(dcp.target) != null;
```

Comment:
Shouldn't target == null be checked within getDeploymentGroupNamed

Reply:
that was about 50/50 decision on my part, so this way "won out" by a hair